src/Assignment-4/assignment4.html

```diff
@@ -6,12 +6,84 @@
 	<meta http-equiv="Pragma" content="no-cache" />
 	<meta http-equiv="Expires" content="0" />
 	<script src="bundle.js" type="text/javascript"></script>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+
 </head>
 
 <body>
 	<div id="container">
 
 	<!-- Start your code from here -->
+	<header>
```

Comment:
Header tag section tag everything should be inside container div. Please properly format your code.